config/config.go

```diff
@@ -349,6 +375,23 @@ func (c *Config) CheckWebserverConfigValues() error {
 	return nil
 }
 
+// CheckForexProviderConfigValues checks forex Provider configuration values
+func (c *Config) CheckForexProviderConfigValues() error {
+	var count int
+	for i := range c.ForexProviders {
+		if c.ForexProviders[i].Enabled == true {
+			if c.ForexProviders[i].APIKey == "Key" {
+				log.Fatal("provider api key not set - please set via your config.json file")
+			}
+			count++
+		}
+	}
+	if count < 1 {
+		log.Println("warning config.go CheckProviderConfigValues() no providers set")
```

Comment:
Moving forward we will need to fatal error if there's no forex providers set (After the free unauthenticated fixer.io service is stopped).

Reply:
Added to trello.